types/index.d.ts

```diff
@@ -0,0 +1,90 @@
+// Type definitions for Chalk
+// Definitions by: Thomas Sauer <https://github.com/t-sauer>
+
+export const enum Level {
+	None = 0,
+	Basic = 1,
+	Extended = 2,
+	TrueColor = 3
```

Comment:
Can we change `Extended` to `Ansi256` here? That's the more correct term; I've never seen `extended` used to refer to the Ansi256 space.